workflows/sars-cov-2-variant-calling/sars-cov-2-consensus-from-variation/.workflowhub.yml

```diff
@@ -0,0 +1,5 @@
+version: "0.1"
+registries:
+- url: "https://workflowhub.eu"
+  project: 33
```

Comment:
Can we work on this ? These numeric ids are really unfortunate.

Reply:
For a while I've played with the idea of allowing names instead of ids for the project and workflow. For projects it's fairly straightforward: just get `/projects` then select by `title`, which is a unique attribute and it's set once and for all ("iwc" in https://workflowhub.eu). For workflows, however, `title` is not unique; moreover, the workflow's name could change in the course of its revision history. This means we might end up having multiple instances of the same workflow if we're not careful.

My proposal to make this manageable is to **set the workflow's name/title to its repository name** (e.g., `parallel-accession-download`, `sars-cov-2-consensus-from-variation`), which is not going to change and should stay unique within the iwc project (unless we mess things up and fail to request a new version somehow).

This involves selecting all workflows that belong to a project, but I had a technical problem while trying that: the `'relationships'` field in the project structure returned by the API does not list workflows (I think it's [this issue](https://github.com/seek4science/seek/issues/447)), so I had to build a reverse map:

```python
def get(endpoint):
    r = session.get(base_url + endpoint)
    r.raise_for_status()
    return r.json()['data']

proj_to_wf = {}
workflows = get("/workflows")
for stub in workflows:
    wf = get(f"/workflows/{stub['id']}")
    for r in wf["relationships"]["projects"]["data"]:
        proj_to_wf.setdefault(r["id"], set()).add(wf["id"])
```

That solves the problem but it's horribly expensive since it requires to individually get all workflows. @fbacall can you help with this?

Reply:
Try again, it should list them now

Reply:
BTW, re: workflow naming, see: iwc-workflows/parallel-accession-download#1